module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -759,6 +774,10 @@ public function login($request)
             // for example):
             $this->getAuth()->preLoginCheck($request);
 
+            // Get the main auth method before switching to any delegate:
+            $mainAuthMethod = $this->getAuth() instanceof ChoiceAuth
```

Comment:
Any reason not to use `getSelectedAuthMethod` here?

Reply:
No, I just fumbled!